### DISCUSSION TITLE
Source 2

### DISCUSSION THREAD

**Discussion by sirherrbatka** [markus-wa/demoinfocs-golang#341] [General]
Hi, 

https://www.youtube.com/watch?v=oIv7m2vhn5g&fbclid=IwAR3hFV1qGS4_TUrbNnAW90RPpGVl7am7NuaR_T0QkkfAlPcKhh5bTkvj7mo

There are rumors that CS:GO source 2 is actually in the works. I wonder what is your opinion on this. Do you think that migrating demoinfocs to source 2 would be substantial effort?

**Comment by markus-wa** [comment]
Hi, good question!

Considering https://github.com/skadistats/clarity works for both CS:GO and Dota2 (which is already on Source 2) I think there either won't be anything to migrate or it will be pretty easy to migrate.

Either way, this library will support Source 2 as soon as possible :slightly_smiling_face:

**Reply by sirherrbatka** [reply]
Yeah, It certainly looks to me like there is no solid reason to play with the dem file format.
